tools/interop_matrix/client_matrix.py

```diff
@@ -18,11 +18,11 @@
 
 def get_github_repo(lang):
     return {
-        'go': 'git@github.com:grpc/grpc-go.git',
-        'java': 'git@github.com:grpc/grpc-java.git',
-        'node': 'git@github.com:grpc/grpc-node.git',
+        'go': 'https://github.com:grpc/grpc-go.git',
+        'java': 'https://github.com:grpc/grpc-java.git',
+        'node': 'https://github.com:grpc/grpc-node.git',
         # all other languages use the grpc.git repo.
-    }.get(lang, 'git@github.com:grpc/grpc.git')
+    }.get(lang, 'https://github.com/grpc/grpc.git')
```

Comment:
Any particular reason for this change?

Reply:
Yes, I face security key touch requests when access via git@github.com scheme. If you only need read access to the repo, https:// is preferable as it does not require credentials.